src/Microsoft.DotNet.Configurer/NuGetCacheSentinel.cs

```diff
@@ -23,7 +23,7 @@ private string NuGetCachePath
             {
                 if (string.IsNullOrEmpty(_nugetCachePath))
                 {
-                    _nugetCachePath = PackageDependencyProvider.ResolvePackagesPath(null, null);
+                    _nugetCachePath = NuGetPathContext.Create(new NullSettings()).UserPackageFolder;
```

Comment:
We are no longer running this for teh -- options above. We are only running this for verbs. But still, we may not have a project.json available (dotnet new). I tried to use the best of what we had at the time. I would love to add in support for the global.json. How do you guys find the global.json? Based on project.json?